### ISSUE TITLE
NTR: West Nile virus RT-PCR in CSF positive (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8453]
New term request West Nile virus RT-PCR in CSF positive (jadhav_091115214854): Cerebrospinal fluid; Reverse transcriptase polymerase chain reaction; RNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000759
- label: Positive CSF West Nile virus nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of West Nile virus nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8453
- synonyms:
